listener/src/event_handler/mod.rs

```diff
@@ -20,6 +20,9 @@ pub use initialization_handler::InitializationHandler;
 mod flux_transition_handler;
 pub use flux_transition_handler::FluxTransitionHandler;
 
+mod sellorder_handler;
```

Comment:
nit: would call the module just order_handler

Reply:
done